src/routes/Settings/SettingsIndexView.js

```diff
@@ -82,7 +83,7 @@ const getSettingsCardProperties = ({
   path: 'settings/sign',
   iconName: 'message',
   searchTags: '',
-  isVisible: isFullMnemonic,
+  isVisible: isFullMnemonic || customType === 'privateKey',
```

Comment:
> isFullMnemonic || customType === 'privateKey'

too verbose...
it is just `!isReadOnly`

Reply:
why we have two properties: `custom Type` and `type`? What's difference between them?